mobile_back_samsung/README.md

```diff
@@ -0,0 +1,10 @@
+# Building MLPerf Open app with Samsung backend
+
+<!-- markdown-link-check-disable-next-line -->
+1. Update the "lib" folder at 'mobile_app_open/mobile_back_samsung/samsung/lib' by copying it from [here](https://github.com/mlcommons/mobile_back_samsung/tree/samsung_backend_flutter_libs/samsung_libs/mobile_back_samsung/samsung)
```

Comment:
I would be helpful if Samsung can provide a checksum for each `*.so` file, which needs to be copied over, to verify its compatibility with the code.

Reply:
Updated the README with .so libs hashes. 
https://github.com/mlcommons/mobile_app_open/blob/submission_v2.1_revert_samsung/mobile_back_samsung/README.md